client/src/main/webjar/organization-details/organization-details.html

```diff
@@ -0,0 +1,145 @@
+<div ng-if="organizationDetails.loading"
+     layout="row"
+     layout-sm="column"
+     layout-align="space-around">
+    <md-progress-circular md-mode="indeterminate">
+    </md-progress-circular>
+</div>
+
+<div ng-if="organizationDetails.orgInfo"
+     layout="column"
+     layout="fill">
+
+    <!-- Toolbar with program actions. -->
+    <md-toolbar class="md-hue-2">
+        <div class="md-toolbar-tools">
+
+            <md-button class="md-icon-button"
+                       aria-label="Back to map"
+                       ui-sref="map">
+                <md-icon md-font-library="material-icons">
+                    arrow_back
+                </md-icon>
+                <md-tooltip md-direction="bottom">
+                    <span>
+                        Back to map
+                    </span>
+                </md-tooltip>
+            </md-button>
+
+            <h2>
+                <span ng-bind="organizationDetails.orgInfo.name">
+                </span>
+            </h2>
+
+            <span flex></span>
+
+            <md-button class="md-icon-button"
+                       aria-label="Delete organization"
+                       ng-click="organizationDetails.deleteOrganization($event)">
+                <md-icon md-font-library="material-icons">
+                    delete
+                </md-icon>
+                <md-tooltip md-direction="bottom">
+                    <span>
+                        Delete organization
+                    </span>
+                </md-tooltip>
+            </md-button>
```

Comment:
Doesn't need to be fixed now (in this PR), but we shouldn't let them try to delete an organization that has programs. I expect the backend won't allow that.